build/build-image/Dockerfile

```diff
@@ -44,13 +44,19 @@ RUN go get -u golang.org/x/tools/cmd/goimports
 # RUN gcloud components update
 RUN gcloud components update && gcloud components install app-engine-go
 
+# the kubernetes version for the file
+ENV KUBERNETES_VER 1.12.10
+
 # overwrite kubectl as we want a specific version
-ENV KUBECTL_VER 1.12.10
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VER}/bin/linux/amd64/kubectl && \
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VER}/bin/linux/amd64/kubectl && \
     chmod go+rx ./kubectl && \
     mv ./kubectl /usr/local/bin/kubectl
 RUN echo "source <(kubectl completion bash)" >> /root/.bashrc
 
+# install the release branch of the code generator tools
+RUN mkdir -p /go/src && cd /go/src && mkdir -p k8s.io && cd k8s.io && \
```

Comment:
I realize it was written this way before, but I don't understand the multiple `mkdir` commands, `mkdir -p` will create intermediate directories so a single `mkdir -p /go/src/k8s.io && cd /go/src/k8s.io \` should do the trick.

Reply:
Oooh! Good point. I'll make this change and then merge.